### PR TITLE
Technigo Week 12 - Project Movies - Malin Göthe & Lars Söderman

### PR DESCRIPTION
**Pull request by M4L5**
Link to my deployed project: 
https://sharp-montalcini-473ded.netlify.app/

My reflections on how this week's project turned out: 
Lars and I worked very well together and we are both happy with the result. 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+export const Home = () => {
+const [ movieList, setMovieList ] = useState([])
+ 
+    const fetchMovieList = () => {
+        fetch(MOVIE_LIST_URL) 
+            .then(res => res.json())
+            .then(json => setMovieList(json.results))
+            .catch(err => console.error(err))
```
Comment (liaback-lowstett) [inline code/src/components/Home.js:15]: No console in production 😉

File: code/src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    useEffect(() => {     
+        fetchMovieList()   
+    }, [])
+    
+    return (
+        <div className="movie-list-container">    
+            {movieList.map(movie => {  
+                return (
+                    <MovieElement {...movie} />
```
Comment (liaback-lowstett) [inline code/src/components/Home.js:26]: Nice  use of spread operator!

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,128 @@
+* {
+  object-fit: cover;
```
Comment (liaback-lowstett) [inline code/src/index.css:2]: Smart to place it here

File: code/src/pages/Details.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    const { id } = useParams()
+
+    const [ details, setDetails ] = useState({})
+    
+    // const fetchMovieDetails = () => {
+    //     fetch(MOVIE_DETAIL_URL(id)) 
+    //         .then(res => res.json())
+    //         .then(json => setDetails(json))
+    //         .catch(err => console.error(err))
+    // }
```
Comment (liaback-lowstett) [inline code/src/pages/Details.js:20]: Remove? 🤔

File: code/src/pages/Details.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+    useEffect(() => {  
+            fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=84c715899a256d0ed1ae1ac98d6fb9a6&language=en-US`)
+                .then(res => res.json())
+                .then(json => setDetails(json))
+                .catch(err => console.error(err))
```
Comment (liaback-lowstett) [inline code/src/pages/Details.js:26]: No console in production 😉

**Review by liaback-lowstett — COMMENTED** [review]
Awesome work! Looks really good 👍 

We noticed that when you realod the detail page you get the message "page not found" , take a look at this solution in Stack-overflow and you might solve it: https://stackoverflow.com/c/technigo/questions/990

/Felicia, Sofia & Caroline

File: code/src/components/Home.js (hunk 1 above)
Comment (KarolCislowski) [reply code/src/components/Home.js:15]: 👮🏻

File: code/src/pages/Details.js (hunk 4 above)
Comment (KarolCislowski) [reply code/src/pages/Details.js:20]: for sure 😸

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Nice project but you need to clean it a little from console outputs and commented out pieces of code. 
You need to work also on your grid:
image

**Review by KarolCislowski — APPROVED** [review]
Great work 👍🏻

**Pull request closed without merging**